cdap-gateway/src/main/java/co/cask/cdap/gateway/router/RouterPathLookup.java

```diff
@@ -186,6 +186,11 @@ private String getV3RoutingService(String [] uriParts, AllowedMethod requestMeth
       // /v3/namespaces/{namespace-id}/data/datasets/{name}/properties
       // /v3/namespaces/{namespace-id}/data/datasets/{name}/admin/{method}
       return Constants.Service.DATASET_MANAGER;
+    } else if (uriParts.length >= 2 && uriParts[1].equals("config")) {
+      // TODO: config endpoint only hits stream service
```

Comment:
You mean the stream handler? So the stream handler is handling /v3/config calls to give the cConf?